src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -5740,5 +6139,226 @@ void M()
+                    {
+                        N(SyntaxKind.IdentifierToken, "y");
+                    }
+                }
+                EOF();
+            }
+        }
+
+        [Fact]
+        public void UnsignedRightShiftAssignment_03()
+        {
+            foreach (var options in new[] { TestOptions.RegularPreview, TestOptions.Regular10, TestOptions.RegularNext })
+            {
+                UsingExpression("x >> >= y", options,
+                    // (1,6): error CS1525: Invalid expression term '>='
+                    // x >> >= y
+                    Diagnostic(ErrorCode.ERR_InvalidExprTerm, ">=").WithArguments(">=").WithLocation(1, 6)
+                    );
+
+                N(SyntaxKind.GreaterThanOrEqualExpression);
+                {
+                    N(SyntaxKind.RightShiftExpression);
+                    {
+                        N(SyntaxKind.IdentifierName);
+                        {
+                            N(SyntaxKind.IdentifierToken, "x");
+                        }
+                        N(SyntaxKind.GreaterThanGreaterThanToken);
+                        M(SyntaxKind.IdentifierName);
+                        {
+                            M(SyntaxKind.IdentifierToken);
+                        }
+                    }
+                    N(SyntaxKind.GreaterThanEqualsToken);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "y");
+                    }
+                }
+                EOF();
+            }
+        }
+
+        [Fact]
+        public void UnsignedRightShiftAssignment_04()
+        {
+            foreach (var options in new[] { TestOptions.RegularPreview, TestOptions.Regular10, TestOptions.RegularNext })
+            {
+                UsingExpression("x >>> = y", options,
+                    // (1,7): error CS1525: Invalid expression term '='
+                    // x >>> = y
+                    Diagnostic(ErrorCode.ERR_InvalidExprTerm, "=").WithArguments("=").WithLocation(1, 7)
+                    );
+
+                N(SyntaxKind.SimpleAssignmentExpression);
+                {
+                    N(SyntaxKind.UnsignedRightShiftExpression);
+                    {
+                        N(SyntaxKind.IdentifierName);
+                        {
+                            N(SyntaxKind.IdentifierToken, "x");
+                        }
+                        N(SyntaxKind.GreaterThanGreaterThanGreaterThanToken);
+                        M(SyntaxKind.IdentifierName);
+                        {
+                            M(SyntaxKind.IdentifierToken);
+                        }
+                    }
+                    N(SyntaxKind.EqualsToken);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "y");
+                    }
+                }
+                EOF();
+            }
```

Comment:
nit: I was trying to brainstorm cases to tests. Scenarios like `is A<B<C<T>>>`, `as A<B<C<T>>>`, `(A<B<C<T>>>, Type) a = ...` are probably already covered (plus those relate to parsing types, not parsing expressions).
So maybe a legit expression scenario: `a < b < c < t >>> n`, or an incremental scenario transitioning from type to expression or vice-versa?